### PR TITLE
Approval service

### PR DESCRIPTION
**Pull request by michellengnx**

### CODE REVIEW

**Comment by Andrewnt219** [comment]
https://github.com/seneca-vas/seneca-course-reviews-admin/pull/5/commits/d21bad4df5ea70a46c85f011daeba32f33f8eedc use `git rebase` next time

File: src/modules/course-reviews/service/course-reviews-service.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import { firestore } from "@lib/firebase";
```
Comment (Andrewnt219) [inline src/modules/course-reviews/service/course-reviews-service.ts:1]: This file does not belong to the admin repo. Please remove it.

File: src/modules/course-reviews/service/index.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export * from "./course-reviews-service"
```
Comment (Andrewnt219) [inline src/modules/course-reviews/service/index.ts:1]: This line does not belong to admin repo.

**Review by Andrewnt219 — CHANGES_REQUESTED** [review]

File: src/modules/course-reviews/service/index.ts (hunk 2 above)
Comment (michellengnx) [reply src/modules/course-reviews/service/index.ts:1]: Removed the line!

File: src/modules/course-reviews/service/course-reviews-service.ts (hunk 1 above)
Comment (michellengnx) [reply src/modules/course-reviews/service/course-reviews-service.ts:1]: Done! Should I keep the model folder if the approval service simply change the isApproved value from false to true, which doesn't really need the CourseReview schema to do so for the time being?

Comment (Andrewnt219) [reply src/modules/course-reviews/service/course-reviews-service.ts:1]: No, just set up an empty service.

**Comment by Andrewnt219** [comment]
@michellengnx don't need to create a new PR next time, just push new commits and the PR is updated.

**Pull request closed without merging**